Model/Orders.php

```diff
@@ -285,14 +294,23 @@ private function getCreditMemos($order)
     public function getOrder($order, ?int $userId)
     {
         $coupon = $order->getCouponCode();
+        $subtotal = $order->getSubtotal();
+        $subtotalInclTax = $order->getSubtotalInclTax();
+        $discount = $order->getDiscountAmount();
+
         return [
             'id' => $order->getId(),
             'order_number' => $order->getIncrementId(),
             'customer_id' => $userId,
             'created_at' => $order->getCreatedAt(),
             'updated_at' => $order->getUpdatedAt(),
             'grand_total' => $order->getGrandTotal(),
-            'subtotal' => $order->getSubtotal(),
+            'subtotal' => $subtotal,
+            'subtotal_including_tax' => $subtotalInclTax,
+            'subtotal_with_discount_excluding_tax' => $subtotal + $discount,
```

Comment:
@damienwebdev Is this arithmetic going to be safe to do like this?

Reply:
I think it's safe, they're only 4 digits of precision, but I don't think we need to do this anyway, I don't think we need them at this point in time, and computation in the resolver is likely a bad precedent to set.